### PR TITLE
Add support of CHUNK_SIZE and ON_DUPLICATE in TS.ADD

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java [hunk 1]
```diff
@@ -246,6 +246,21 @@ public void testAdd() {
       Assert.assertEquals(600L, client.add("seriesAdd", 600L, 1.1, 10000, null));
     }
 
+    Assert.assertEquals(
+        400L,
```
Comment (filipecosta90) [inline src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:250]: Can we test for a duplicate timestamp? I believe 400L is not a duplicate one correct?

File: src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java [hunk 2]
```diff
@@ -246,6 +246,21 @@ public void testAdd() {
...
+        client.add(
+            "seriesAdd4",
+            400L,
+            0.4,
+            7898L /*retentionTime*/,
+            true,
+            1000L,
+            DuplicatePolicy.MAX,
+            labels));
```
Comment (sazzad16) [inline src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:259]: @gkorland @filipecosta90 Would you please re-evaluate google formatter? This is making the code way too long.

File: src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java [hunk 3]
```diff
@@ -404,7 +405,46 @@ public long add(
...
+      byte[][] args =
+          tsAddArgs(
+              sourceKey,
+              timestamp,
+              value,
+              retentionTime,
+              uncompressed,
+              chunkSize,
+              duplicatePolicy,
+              labels);
```
Comment (sazzad16) [inline src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java:447]: @gkorland @filipecosta90 Would you please re-evaluate google formatter? This is making the code way too long.

File: src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java (hunk 1 above)
Comment (sazzad16) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:250]: The entire `seriesAdd4` key is new here. 
Also, I did not find any duplicate timestamp example to follow/get an idea.

Comment (filipecosta90) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:250]: @sazzad16 example of on_duplicate in ts.add

# using cli

```
redis-cli ts.add serie1 1 10.0
redis-cli ts.add serie1 1 5.0 ON_DUPLICATE MAX
redis-cli ts.range serie1 - +
```
Expected result:
```
$ redis-cli ts.add serie1 1 10.0
(integer) 1
$ redis-cli ts.add serie1 1 5.0 ON_DUPLICATE MAX
(integer) 1
$ redis-cli ts.range serie1 - +
1) 1) (integer) 1
   2) 10
```

# examples of go client
https://godoc.org/github.com/RedisTimeSeries/redistimeseries-go#example-Client-AddWithOptions-DuplicateDatapointsMaxDuplicatePolicy
https://godoc.org/github.com/RedisTimeSeries/redistimeseries-go#example-Client-AddWithOptions-DuplicateDatapointsLastDuplicatePolicy

File: src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java (hunk 2 above)
Comment (filipecosta90) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:259]: @sazzad16 any formatter will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.

Comment (filipecosta90) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java]: Let's discuss this on a specific PR. Any format will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.

File: src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java (hunk 3 above)
Comment (filipecosta90) [reply src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java:447]: Let's discuss this on a specific PR. Any format will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.

File: src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java (hunk 1 above)
Comment (sazzad16) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:250]: @filipecosta90 I've added one before seeing your reply. Could you please check that version?

Comment (filipecosta90) [reply src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java:250]: :+1:

**Review by filipecosta90 — APPROVED** [review]

**Pull request merged**